lib/Mage/System/Ftp.php

```diff
@@ -373,7 +373,7 @@ public function nlist($dir = "/")
      *
      * @param string $dir
      * @param bool $recursive
-     * @return mixed
+     * @return array
```

Comment:
https://www.php.net/manual/en/function.ftp-rawlist.php
> Returns an array where each element corresponds to one line of text. Returns false when passed directory is invalid. 
```suggestion
     * @return string[]|false
```